projects/architecture/interfaces/primitives/transactions.proto

```diff
@@ -1,15 +1,36 @@
 syntax = "proto3";
 
+import "./addresses.proto";
+
 message Transaction {
-  uint32 version = 1;
-  string sender = 2;
-  string contractAddress = 3;
-  string payload = 4;
-  string signature = 5;
+  TransactionHeader header = 1;
+  TransactionBody body = 2;
+  // bytes signature = 3;
 }
 
-message TransactionAppendix {
+message TransactionHeader {
   uint32 version = 1;
-  repeated string prefetchAddresses = 2;
-  string subscriptionKey = 3;
+  UniversalAddress sender = 2;
+  // TransactionLifespan lifespan = 3;
```

Comment:
Why have you decided to not address lifespan/ttl here? If it's going to be part of a different PR, could you remove this commented out code from here for now?

Reply:
It's a placeholder. It was a reasonable compromise since I do aim to implement it very soon